docs/integrations/web-servers/opentelemetry/iis-10-opentelemetry.md

```diff
@@ -0,0 +1,203 @@
+
+-   IIS Access Logs (W3C default format) Sumo Logic expects logs in [W3C](https://docs.microsoft.com/en-us/windows/desktop/http/w3c-logging) format with following fields:
+```sql
+#Fields: date time s-ip cs-method cs-uri-stem cs-uri-query s-port cs-username c-ip cs(User-Agent) cs(Referer) sc-status sc-substatus sc-win32-status time-taken
+```
+IIS allows you to choose fields to log in IIS access logs. For explanations on the various fields and their significance see this [link](https://docs.microsoft.com/en-us/windows/desktop/http/w3c-logging).
+
+-   HTTP Error Logs Sumo Logic expects Error logs in following format :
+```sql
+#Fields: date time c-ip c-port s-ip s-port protocol_version verb cookedurl_query protocol_status siteId Reason_Phrase Queue_Name
+```
+
+For information on how to configure HTTP Error Logs, and for explanations on the various HTTP Error Log fields and their significance see this [link](https://support.microsoft.com/en-us/help/820729/error-logging-in-http-apis).
+
+## Enable logging on your IIS Server.
+
+Perform the following task, if logging on your IIS Server is not already enabled. To enable logging on your IIS Server, do the following:
+
+1.  Open IIS Manager.
+2.  Select the site or server in the Connections pane, then double-click Logging.
+
+Enhanced logging is only available for site-level logging. If you select the server in the Connections pane, then the Custom Fields section of the W3C Logging Fields dialog is disabled.
+
+1.  In the Format field under Log File, select W3C and then click Select Fields. IIS App works on default fields selection.
+2.  Select following fields, if not already selected. Sumo Logic expects these fields in IIS logs for the IIS App to work by default:
+```sh
+date time s-ip cs-method cs-uri-stem cs-uri-query s-port cs-username c-ip cs(User-Agent) cs(Referer) sc-status sc-substatus sc-win32-status time-taken
+```
+
+For more information about IIS log format and log configuration refer [link](https://docs.microsoft.com/en-us/iis/get-started/whats-new-in-iis-85/enhanced-logging-for-iis85).
+
+Once the logs are configured to be written into a local file follow the below step to configure collection in sumo.
+
+## Collection Configuration & App installation
+
+As part of the setting up the collection process and app installation user can select the App from App Catalog and click on Install App. Please follow the steps below:
+
+### Step 1: Set up Collector
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the "Use an existing Collector" option.
+:::
+
+If you want to use an existing Otel Collector then this step can be skipped by selecting the option of using an existing Collector.
+
+If you want to create a new Collector please select **Add a new Collector** option.
```

Comment:
To create a new Collector:

1. Select the **Add a new Collector** option.
1. Select the platform for which you want to install the Sumo OpenTelemetry Collector.